cucumber-expressions/java/src/test/java/io/cucumber/cucumberexpressions/ExpressionExamplesTest.java

```diff
@@ -40,7 +40,7 @@ public ExpressionExamplesTest(String expressionString, String text, String expec
         String[] chunks = s.split("---");
         for (String chunk : chunks) {
             chunk = chunk.trim();
-            data.add(chunk.split("\n"));
+            data.add(chunk.split(System.lineSeparator()));
```

Comment:
Please ensure the examples.txt file has unix line endings and this shouldn't be needed.

Reply:
Looks good - I’ll merge this when the other implementations have a similar fix.

Reply:
@mpkorstanje, I use Windows as default OS. And I've faced a problem when tried to run tests that were in your repo. I think Files.readAllBytes(Path p) adds line separator depending on the OS that it's executed on.
So I can't use unix line endings if I use Windows. I guess, it won't be convenient for other users too.

Reply:
@aslakhellesoy, I don't really understand what exactly do you mean by "other implementations".

Reply:
JavaScript, Go and Ruby